cmd/cosign/cli/verify/verify_blob_test.go

```diff
@@ -93,3 +95,45 @@ func TestSignaturesBundle(t *testing.T) {
 		t.Fatalf("unexpected encoded signature, expected: %s got: %s", b64sig, gotb64Sig)
 	}
 }
+
+func TestIsIntotoDSSEWithEnvelopes(t *testing.T) {
+	tts := []struct {
+		envelope     dsse.Envelope
+		isIntotoDSSE bool
+	}{
+		{
+			envelope: dsse.Envelope{
+				PayloadType: "application/vnd.in-toto+json",
+				Payload:     base64.StdEncoding.EncodeToString([]byte("This is a test")),
+				Signatures:  []dsse.Signature{},
+			},
+			isIntotoDSSE: true,
+		},
+	}
+	for _, tt := range tts {
+		envlopeBytes, _ := json.Marshal(tt.envelope)
+		got, _ := isIntotoDSSE(envlopeBytes)
+		if got != tt.isIntotoDSSE {
+			t.Fatalf("unexpected envelope content")
+		}
+	}
+}
+
+func TestIsIntotoDSSEWithBytes(t *testing.T) {
+	tts := []struct {
+		envelope     []byte
+		isIntotoDSSE bool
+	}{
+		{
+			envelope:     []byte("This is no valid"),
```

Comment:
could you add a test with a regular signature in here just to make sure it still fails?

Reply:
Oki doki. I think I did what you wanted, my apologies if I misunderstood.